circle.yml

```diff
@@ -1,6 +1,11 @@
 machine:
   node:
     version: 6.2.1
+
+dependencies:
+  post:
+    - npm install react react-native @accounts/common @accounts/client
```

Comment:
Hmm not sure.. devDependencies has version on them so I don't like the fact that it's sitting there as a duplicates.. Although it will make it so when a developer installs this package locally, the lint will pass without a problem so I guess you are right.